feature-libs/product-configurator/rulebased/styles/_configurator-overview-form.scss

```diff
@@ -65,6 +82,9 @@
       }
 
       border: solid 1px var(--cx-color-light);
+      background-color: var(--cx-color-background);
```

Comment:
Also those changes need to be documented

Reply:
Done